glite-info-update-endpoints.spec

```diff
@@ -1,13 +1,27 @@
-Name:		glite-info-update-endpoints
-Version:	3.0.1
-Release:	1%{?dist}
-Summary:	Updates LDAP endpoins for EGI
-Group:		Development/Libraries
-License:	ASL 2.0
-URL:		https://github.com/EGI-Foundation/glite-info-update-endpoints
-Source:		%{name}-%{version}.src.tgz
-BuildArch:	noarch
-BuildRoot:	%{_tmppath}/%{name}-%{version}-build
+%if 0%{?el8}
+%global __python %{__python3}
+%global python_version_prefix 3
+%endif
+
+Name:             glite-info-update-endpoints
+Version:          3.0.1
```

Comment:
maybe better to move to a new minor? like 3.1.0?

Reply:
or at least should be aligned to the version bump . 3.02

Reply:
yup, 3.0.2 at least